src/server/controllers/changePassword.ts

```diff
@@ -172,13 +172,24 @@ const changePasswordInOkta = async (
   res: ResponseWithRequestState,
   successRedirectPath: RoutePaths,
 ) => {
-  const { stateToken } = readEncryptedStateCookie(req) ?? {};
```

Comment:
is `stateToken` being used anymore? Can we remove it from the encrypted state cookie as a result?

Reply:
Good point, have removed it from the encrypted state cookie